### ISSUE TITLE
Relative Url / Container behind https nginx - Network tab contains wrong references

### ISSUE THREAD

**Issue by pylight** [sameersbn/docker-gitlab#326]
Hi,
first of all thanks for the create docker-image :+1: 

I have set up gitlab to run on the relative url **/gitlab** and its running behind a nginx https webserver.
After setting up the container, I have one problem:

In the *network tab* of projects, the user-avatar source is not linked correctly. It links to /gitlab/gitlab/<imageName>.png instead of /gitlab/<imageName>.png (where the image exists):

![example](http://i.imgur.com/O7GuDIf.png)

I don't know if this is an webserver/configuration issue, an issue of the container or of gitlab itself. Strangely I've also set an avatar for my user but gitlab tries to link to the no-avatar image. If I click on a certain commit in the network tab, the commit view shows my correct avatar with the correct path.

**docker-compose:**
```
postgres:
  image: localhost:5000/postgres
  environment:
    - POSTGRES_USER=gitlab
    - POSTGRES_PASSWORD=somePass
  ports:
    - "127.0.0.1:5432:5432"
redis:
  image: localhost:5000/redis
  ports:
    - "127.0.0.1:6379:6379"
service:
  image: localhost:5000/gitlab
  links:
    - redis:redisio
    - postgres:postgresql
  ports:
    - "127.0.0.1:11080:80"
    - "11022:22"
  environment:
    - TZ=Europe/Berlin
    - SMTP_ENABLED=true
    - SMTP_DOMAIN=www.host.net
    - SMTP_HOST=some.host.net
    - SMTP_PORT=25
    - GITLAB_TIMEZONE=Berlin
    - GITLAB_HOST=some.host.net
    - GITLAB_HTTPS=true
    - GITLAB_PORT=443
    - GITLAB_SSH_PORT=11022
    - GITLAB_RELATIVE_URL_ROOT=/gitlab
    - GITLAB_BACKUPS=daily
    - GITLAB_BACKUP_TIME=01:00
    - GITLAB_GRAVATAR_ENABLED=false
    - DB_TYPE=postgres
    - DB_NAME=gitlab
    - DB_USER=gitlab
    - DB_PASS=somePass
  volumes:
    - /var/run/docker.sock:/run/docker.sock
    - /usr/bin/docker:/bin/docker
    - /data/gitlab/:/home/git/data/
    - /data/gitlab/logs/:/var/log/gitlab
```

**nginx configuration on host**:
```
# http redirection
server {
    listen       80;
    return 301 https://$host$request_uri;
}

# https configuration
server {
    listen 443;
    server_name host.net;

    ssl_certificate       /etc/path/to/cert.crt;
    ssl_certificate_key   /etc/path/to/cert.key;

    ssl on;
    ssl_session_cache  builtin:1000  shared:SSL:10m;
    ssl_protocols  TLSv1 TLSv1.1 TLSv1.2;
    ssl_ciphers HIGH:!aNULL:!eNULL:!EXPORT:!CAMELLIA:!DES:!MD5:!PSK:!RC4;
    ssl_prefer_server_ciphers on;

    access_log            /var/log/nginx/nginx.access.log;
    client_max_body_size 1G;

    location / {
       root /var/www/info;
       index index.html index.htm;
    }

    location /gitlab/ {
      proxy_set_header        Host $host;
      proxy_set_header        X-Real-IP $remote_addr;
      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
      proxy_set_header        X-Forwarded-Proto $scheme;

      proxy_pass              http://127.0.0.1:11080;
      proxy_redirect          off;
      proxy_read_timeout      180;
      }
}
```

(I changed hostname, passwords and paths for privacy reasons in the configurations above)
Did I configure something wrong?
Thanks :)

**Comment by sameersbn** [comment]
@pylight there has been an issue with emojis, etc. when using gitlab under a relative url. I don't know if this is a configuration issue with the image or an issue with gitlab itself.

To check if this is an image specific issue or not, I need to setup gitlab manually,  I have not found sufficient to do this. I hope to look into this as soon as I find the time.

**Comment by sameersbn** [comment]
@pylight I tested this today on gitlab installed from source on a virtual machine and found that this is a gitlab issue. We need to report this to gitlab upstream.

**Comment by pylight** [comment]
Thanks for investigation. I've created an issue for [gitlab ce](https://gitlab.com/gitlab-org/gitlab-ce/issues/1728).

**Comment by sameersbn** [comment]
fixed in release `8.0.0`
